tools/minibrowser/build.gradle

```diff
@@ -1,8 +1,20 @@
+buildscript {
+    repositories {
+        google()
+    }
+
+    dependencies {
+        classpath 'androidx.navigation:navigation-safe-args-gradle-plugin:2.5.2'
+    }
+}
+
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
 }
 
+apply plugin: 'androidx.navigation.safeargs.kotlin'
```

Comment:
You can load and add it at once in plugins.id

Reply:
true

Reply:
fixing